Various errors and bugs when rotating from landscape to portrait

### Steps to reproduce

1. Select auto or landscape as "screen rotation" in general settings.
2. Turn on automatic fullscreen option for Player. or don't open
3. Hold the phone horizontally
4. Play downloaded video online
5. Rotate the phone vertically.
6. Error

Or if "screen rotation" landscape is selected, the same error occurs when holding the phone vertically and playing the video

### Expected behavior

There is no error when the "local version of video" message rotates vertically if "Screen rotation" is set to auto

### Actual behavior

https://github.com/user-attachments/assets/27eb786b-b106-4caa-8574-63388491b36c

### LibreTube version

0.26.0 (58)

### Android version

Android

### Other details

kotlin.UninitializedPropertyAccessException: lateinit property playerController has not been initialized
	at com.github.libretube.ui.fragments.PlayerFragment.restartActivityIfNeeded(PlayerFragment.kt:1500)
	at com.github.libretube.ui.fragments.PlayerFragment.onConfigurationChanged(PlayerFragment.kt:1530)
	at androidx.fragment.app.Fragment.performConfigurationChanged(Fragment.java:3246)
	at androidx.fragment.app.FragmentManager.dispatchConfigurationChanged(FragmentManager.java:3375)
	at androidx.fragment.app.FragmentManager.lambda$new$0$androidx-fragment-app-FragmentManager(FragmentManager.java:607)
	at androidx.fragment.app.FragmentManager$$ExternalSyntheticLambda0.accept(D8$$SyntheticClass:0)
	at androidx.activity.ComponentActivity.onConfigurationChanged(ComponentActivity.kt:849)
	at androidx.appcompat.app.AppCompatActivity.onConfigurationChanged(AppCompatActivity.java:229)
	at com.github.libretube.ui.activities.MainActivity.onConfigurationChanged(MainActivity.kt:543)
	at android.app.ActivityThread.performActivityConfigurationChanged(ActivityThread.java:6544)
	at android.app.ActivityThread.performConfigurationChangedForActivity(ActivityThread.java:6424)
	at android.app.ActivityThread.handleActivityConfigurationChanged(ActivityThread.java:6794)
	at android.app.servertransaction.ActivityConfigurationChangeItem.execute(ActivityConfigurationChangeItem.java:55)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8669)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)

**Other important error**

**Other error** 

You should do this quickly before the video opens.
1**. Select auto or landscape as  screen rotation. Enable auto rotation on the device
2. Turn on Auto fullscreen option for Player. Or don't open it.
3. Keep the device horizontal.
4. Play any video
5. Rotate device to portrait orientation. Do this quickly.
6. Error 2


**java.lang.RuntimeException: Unable to create service com.github.libretube.services.VideoOnlinePlayerService: java.lang.IllegalStateException: Session ID must be unique. ID=com.github.libretube.services.VideoOnlinePlayerService
	at android.app.ActivityThread.handleCreateService(ActivityThread.java:4953)
	at android.app.ActivityThread.access$1800(ActivityThread.java:310)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2300)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8669)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
Caused by: java.lang.IllegalStateException: Session ID must be unique. ID=com.github.libretube.services.VideoOnlinePlayerService
	at androidx.media3.session.MediaSession.<init>(MediaSession.java:686)
	at androidx.media3.session.MediaLibraryService$MediaLibrarySession.<init>(MediaLibraryService.java:679)
	at androidx.media3.session.MediaLibraryService$MediaLibrarySession$Builder.build(MediaLibraryService.java:659)
	at com.github.libretube.services.AbstractPlayerService.createPlayerAndMediaSession(AbstractPlayerService.kt:305)
	at com.github.libretube.services.AbstractPlayerService.onCreate(AbstractPlayerService.kt:246)
	at android.app.ActivityThread.handleCreateService(ActivityThread.java:4940)
	... 9 more**




**Another bug**: 
Screen rotation: auto or landscape
1. Turn on automatic full screen mode.
2. Hold the device horizontally.
3.  Play the video. 
4. Rotate the device vertically while the video is playing.
5. Play button state does not change

https://github.com/user-attachments/assets/6e4d7cf9-615d-4020-9929-7600ec75043c

**Another bug 2**
Screen rotation: auto or landscape
1. Turn off automatic fullscreen option for Player. 
2. Hold the phone horizontally and play the video.
3. Rotate device vertically while video is progressing
4. MediaCondecRendered error.


https://github.com/user-attachments/assets/0cb65074-3566-468c-94e4-949191b20187

    
    

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [x] I have written a short but informative title.
- [x] I will fill out all of the requested information in this form.

--- a/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt
+++ b/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt
@@ -1495,15 +1495,12 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
         val orientation = resources.configuration.orientation
         if (commonPlayerViewModel.isFullscreen.value != true && orientation != playerLayoutOrientation) {
             // remember the current position before recreating the activity
-            arguments?.putLong(
-                IntentData.timeStamp,
-                playerController.currentPosition / 1000
-            )
             playerLayoutOrientation = orientation
 
             viewModel.isOrientationChangeInProgress = true
 
-            playerController.release()
+            if (::playerController.isInitialized) playerController.release()
+
             activity?.recreate()
         }
     }

